src/app.js

```diff
@@ -1,11 +1,14 @@
+
 import express from 'express';
 import auth from './routes/auth.route';
+import protect from './routes/index';
```

Comment:
Because I made a middleware to check if a user is signed in, I had to create a dummy protected route to test the middleware. When we have got some protected route, we will remove it.